docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Health.adoc

```diff
@@ -136,15 +136,29 @@ If a deployment does not provide any readiness checks, WildFly will automaticall
 [NOTE]
 ====
 This allows applications that does not provide readiness checks to still be able to inform cloud containers when they are ready to serve requests.
-Setting empty-readiness-checks-status` to `DOWN` ensures that the server will not be ready until the application is deployed. At that time, the `ready-<deployment name>`
-will be added (which returns `UP`) and the `empty-readiness-checks` procedure will no longer be checked as there is no a readiness check procedure provided by the deployment.
+Setting `empty-readiness-checks-status` to `DOWN` ensures that the server will not be ready until the application is deployed. At that time, the `ready-<deployment name>`
+will be added (which returns `UP`) and the `empty-readiness-checks` procedure will no longer be checked as there is now a readiness check procedure provided either by the deployment or by the server.
 ====
 
 WildFly also provide a liveness procedure that is checked to determine if the application server is live:
 
 * `empty-liveness-checks` determines the status when there are no liveness check procedures deployed to the server. The outcome of this procedure is determined by the `empty-liveness-checks-status` attribute. If the attribute is
 `UP` (by default), the server can be live when there are no liveness checks in the deployments.  Setting the `empty-liveness-checks-status` attribute to `DOWN` will make this procedure fail when there are no liveness checks in the deployments.
 
+* `empty-startup-checks` determines the status when there are no startup check procedures deployed to the server. The outcome of this procedure is determined by the `empty-startup-checks-status` attribute. If the attribute is
```

Comment:
Maybe there could be a title for this section, similarly to what has been done for [readiness](https://github.com/wildfly/wildfly/pull/14400/files#diff-38436541a0ba0a2d1844aebdaa5e6efd36fe778bac666d4c9545fa6018b5aabfR126) and [liveness](https://github.com/wildfly/wildfly/pull/14400/files#diff-38436541a0ba0a2d1844aebdaa5e6efd36fe778bac666d4c9545fa6018b5aabfR143) check sections above.
WDYT?

Reply:
this is all part of the section `Default Server Procedures`. I've just added enpty-startup-checks bullet. Note that there is also empty-liveness and 3 readiness checks so I think this is better left together.

Reply:
The change you've just pushed fixes it perfectly. Fill free to resolve this thread.